src/main/java/org/takes/facets/auth/PsByFlag.java

```diff
@@ -98,19 +97,22 @@ public PsByFlag(final String flg, final Map<Pattern, Pass> map) {
     }
 
     @Override
-    public Iterator<Identity> enter(final Request req) throws IOException {
+    public Opt<Identity> enter(final Request req) throws IOException {
         final Iterator<String> flg = new RqHref.Base(req).href()
             .param(this.flag).iterator();
-        final Collection<Identity> users = new ArrayList<Identity>(1);
+        Opt<Identity> user = null;
```

Comment:
@antonini we don't use NULL anywhere in the project, see http://www.yegor256.com/2014/05/13/why-null-is-bad.html